app/src/main/java/com/example/karam/LandingPage.java

```diff
@@ -0,0 +1,82 @@
+package com.example.karam;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.ListView;
+import android.widget.Toast;
+
+public class LandingPage extends AppCompatActivity {
+    private static String[] languages = {"English",
```

Comment:
same comment as previous files

Reply:
If this is used in more than one file, pull it out into a new file and import that everywhere

Reply:
These are dummy values that will be removed when I add code for business logic.